### PR TITLE
Enable Galaxy file display (test and descriptor)

### PR DESCRIPTION
**Pull request by denis-yuen**
* enables file descriptor and test file display en route to Galaxy DAG display #3174
* consider as feedback for https://github.com/dockstore/dockstore/issues/3154 (i.e. should reduce number of places we need to touch to add new languages to the UI)

Will need cleanup, but need to get unblocked to work on DAG hookup

Can be reverted after https://github.com/dockstore/dockstore-ui2/pull/918 is cleaned up and goes in, but I'd like to get this in to start trying out workflows from https://github.com/dockstore/dockstore/issues/3186 on dev.dockstore.net

### CODE REVIEW

**Comment by denis-yuen** [comment]
Attaching screenshots of settings to reproduce results

Screenshot from 2020-02-18 12-16-02
Screenshot from 2020-02-18 12-16-04
Screenshot from 2020-02-18 12-16-06
Screenshot from 2020-02-18 12-16-10

File: src/app/workflow/tool-tab/tool-tab.service.ts [hunk 1]
```diff
@@ -46,6 +48,8 @@ export class ToolTabService {
         return 'task\xa0ID';
       case ToolDescriptor.TypeEnum.NFL:
         return 'process\xa0name';
+      case ToolDescriptor.TypeEnum.GXFORMAT2:
+        return 'tool\xa0ID';
```
Comment (agduncan94) [inline src/app/workflow/tool-tab/tool-tab.service.ts]: Comment on weird escape.

Comment (denis-yuen) [reply src/app/workflow/tool-tab/tool-tab.service.ts]: FYI, was copied from CWL case. 
Discussed in person whether it is more appropriate to call this steps or tools (they seem to be used interchangeably looking together)

**Comment by denis-yuen** [comment]
```
ERROR in src/app/configuration.service.ts:38:61 - error TS2339: Property 'bdCatalystSevenBridgesImportUrl' does not exist on type 'Config'.

38     Dockstore.BD_CATALYST_SEVEN_BRIDGES_IMPORT_URL = config.bdCatalystSevenBridgesImportUrl;
                                                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/app/configuration.service.ts:39:53 - error TS2339: Property 'bdCatalystTerraImportUrl' does not exist on type 'Config'.

39     Dockstore.BD_CATALYST_TERRA_IMPORT_URL = config.bdCatalystTerraImportUrl;
```

@coverbeck  any idea actually? I seem to have picked up something from that last merge from develop even though it was a clean merge

**Comment by coverbeck** [comment]
> @coverbeck any idea actually? I seem to have picked up something from that last merge from develop even though it was a clean merge

Hmm, what if you change package.json to point to 1.8.1? Although rc should have it too.

**Comment by denis-yuen** [comment]
> Hmm, what if you change package.json to point to 1.8.1? Although rc should have it too. Did you run `npm run prebuild`?

I did not, but is that necessary anymore? `prebuild` doesn't show up in the readme https://github.com/dockstore/dockstore-ui2#build 
Was under the impression it was run by `npm run build` which is

**Comment by garyluu** [comment]
Going to make a PR for galaxy without mentioning galaxy in code

**Pull request merged**